synapse/lib/certdir.py

```diff
@@ -96,8 +96,16 @@ def _saveCertTo(self, cert, *paths):
         if os.path.isfile(path):
             raise s_common.DupFileName(path=path)
 
-        with s_common.genfile(path) as fd:
-            fd.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert))
+        data = None
+        if isinstance(cert, crypto.X509):
```

Comment:
With the existing difference between _saveCertTo and _savePkeyTo, we should probably just create _savePkcs12To so we dont have to do isinstance() stuff on an argument. ( always gives me the heebie jeebies... )